backend/support/admin.py

```diff
@@ -149,32 +149,55 @@ class LogEntryAdmin(SupportAdmin):
     )
 
     def staff_user(self, obj):
+        """The staffuser."""
         return obj.user.email
 
     def record_affected(self, obj):
+        """The record associated with the staffuser's action."""
         return obj.object_repr
 
     def event(self, obj):
+        """Shares the action taken by the staffuser."""
+
         if obj.action_flag == ADDITION:
             return "Created"
         elif obj.action_flag == DELETION:
             return "Deleted"
         elif obj.action_flag == CHANGE:
             res = "Updated"
             if obj.change_message:
-                _json = json.loads(obj.change_message)[0]
-                if "changed" in _json:
-                    res = "Updated\n"
-                    for field in _json["changed"]["fields"]:
-                        res += f"\n- {field}"
+                _json = json.loads(obj.change_message)
+
+                # The LogEntry has recorded changes.
+                if len(_json) > 0:
+                    _json = _json[0]
+                    if "changed" in _json:
+                        res = "Updated\n"
+                        for field in _json["changed"]["fields"]:
+                            res += f"\n- {field}"
+
+                # No changes were actually made.
+                else:
+                    res += "\n- No changes"
             return res
         return "-"
 
     def content(self, obj):
+        """The raw contents of the record that was changed."""
+
         if obj.change_message:
-            _json = json.loads(obj.change_message)[0]
-            if "content" in _json:
-                return _json["content"]
+            _json = json.loads(obj.change_message)
+
+            # The LogEntry has recorded changes.
+            if len(_json) > 0:
```

Comment:
(suggestion) I believe `if _json:` is more pythonic and slightly more performant than `if len(_json) > 0:`